zebra/zebra_fpm_netlink.c

```diff
@@ -299,7 +298,7 @@ netlink_route_info_fill (netlink_route_info_t *ri, int cmd,
           || (cmd == RTM_DELROUTE
               && CHECK_FLAG (nexthop->flags, NEXTHOP_FLAG_FIB)))
         {
-          netlink_route_info_add_nh (ri, nexthop, recursing);
+          netlink_route_info_add_nh (ri, nexthop, (nexthop->rparent ? 1 : 0));
```

Comment:
Can't netlink_route_info_add_nh figure out the recursing or not?  Do we need a third parameter at this time anymore?